mailserver/mailserver.go

```diff
@@ -352,11 +363,11 @@ func (s *WMailServer) SyncMail(peer *whisper.Peer, request whisper.SyncMailReque
 
 	requestID := fmt.Sprintf("%d-%d", time.Now().UnixNano(), rand.Intn(1000))
 
-	syncRequestsMeter.Mark(1)
+	requestsCounter.WithLabelValues("sync").Inc()
```

Comment:
This should be separate. `syncReqyestsCounter`.

Reply:
Done.